src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -116,6 +121,10 @@ public static LoanPolicy unknown(String id) {
           errors.add(loanPolicyValidationError(CAN_NOT_RENEW_ITEM_ERROR));
           return failedValidation(errors);
         }
+        if (isFixed(getLoansPolicy()) && hasAlternateRenewalLoanPeriodForHolds()) {
+          return failed(new ServerErrorFailure(FIXED_POLICY_HAS_ALTERNATE_PERIOD));
```

Comment:
Can the same be done for alternate renewal period (without holds)?

Reply:
I'm not sure if it is really required, because presence of the field does not give any affect on the renewal logic with fixed policy. And that's something that have to be handled on policy creation step. 

Does it make sense?

Reply:
I don't think I follow.

A check was introduced that reported a server error when the policy is fixed and the alternate renewal period when hold requests are present is defined.

My understanding is that we also chose to block the combination of fixed and alternate renewal period. Why wouldn't we want to report a similar error for this situation?

Reply:
Because previously it was allowed to specify alternate period for holds for fixed policies. And because of this, some data in DB can have such a field, so we should report the error, just to let someone know that the policy should be modified.

But we did not allow renewal period for fixed policies before, therefore no reason for such validation in renewal logic and such violation have to be handled on policy creation step.

Reply:
> But we did not allow renewal period for fixed policies before, therefore no reason for such validation in renewal logic and such violation have to be handled on policy creation step.

I don't think I follow, are you referring to the UI stopping this from happening?

As I understanding it, [this check](https://github.com/folio-org/mod-circulation-storage/compare/CIRCSTORE-159-disallow-periods-for-fixed#diff-d8b0f6ea5a1498408ebca2f2612d4847R15) in your other pull request is the validation to stop this scenario. Have I misunderstood?

Even with those checks, there is no restriction in the interface, and hence, an implementation could allow this. To me, this is the same as the other check that you put in place, what am I missing?

Reply:
@marcjohnson-kint, yep, I meant UI. 

Ok, if you think we should add such a check in renewals logic, then I will proceed with this.